uvtools/version.py

```diff
@@ -5,10 +5,11 @@
 
 from __future__ import absolute_import, division, print_function
 
-import os
+import os, sys
```

Comment:
these should be on separate lines per PEP8

Reply:
Yup my bad. Fixed now.